src/components/TodoList.jsx

```diff
@@ -26,17 +31,25 @@ const TodoList = (props = (todos, isLoading, setIsLoading)) => {
                             (
                                 <div>
                                     {props.todos.map(todoItem => (
-                                        <div key={todoItem.id} className="flex justify-between space-x-5">
+                                        <div key={todoItem.id}
+                                             className="flex justify-between space-x-5 border-solid border-2 border-indigo-600 my-2">
                                             <p>{todoItem.title}</p>
+                                            <p>{todoItem.description}</p>
+                                            <p>{todoItem.priority}</p>
                                             <button
                                                 className="bg-red-500 text-white font-medium px-2 py-1 rounded -x1 hover:bg-red-900"
```

Comment:
You can check if IntelliJ has some tailwind plugin as it has for VSCode to have autocomplete for classes and error messages for classes that don't exist like `-x1`